Pod/Classes/ViewControllers/ARBidFlowViewController.m

```diff
@@ -15,7 +15,7 @@ - (instancetype)initWithArtworkID:(NSString *)artworkID saleID:(NSString *)saleI
                           @"intent": (intent == ARBidFlowViewControllerIntentBid ? @"bid" : @"register")
   };
   if ((self = [super initWithEmission:nil
-                           moduleName:@"BidFlow"
+                           moduleName:@"BidderFlow"
```

Comment:
Will this rename require a change in eigen or anything?

Reply:
At the moment I don't think so. This is internal to how the Eigen Pod connects to React Native components, and I picked the name change because `BidFlow` was a counterpart to `RegistrationFlow`, and they are both Bidder Flows. I think it would be good to change this file to `ARBidderFlowViewController.m` as well.

Reply:
Now I'm thinking maybe this will change things - I'm assuming eigen will need to reinstall the pod. If so we can keep the existing name and make a small followup ticket to update them together.